client/styles/global.scss

```diff
@@ -133,6 +133,47 @@ a {
 	}
 }
 
+#forgotpasswordsubmitbutton {
+	width: 40%;
+	display: block;
+	height: 30px;
+	margin: 0px auto;
+	background-color: #2ecc71;
+	margin-top: 20px;
+	margin-bottom: 35px;
+	border-radius: 5px;
+	cursor: pointer;
+	text-align: center;
+	font-weight: 600;
+	color: white;
+	padding-top: 5px;
+
+  &:hover {
+		background-color: #4cd785;
+	}
+}
+
+#resetpasswordsubmitbutton {
+  width: 40%;
```

Comment:
indentation

Reply:
@peter-hank I have made some changes for indentation. Please review them and if there are any more suggestions then do let me know

Reply:
@singhsanket143 can you use some online tool like http://www.calluna-software.com/Formatter/Css and fix the whole file?

Reply:
@peter-hank Updated the indentation

Reply:
@singhsanket143 it's still not nice, can you make sure the whole file looks good?

Reply:
@peter-hank I have used https://www.freeformatter.com/css-beautifier.html as a tool to indent the whole file. Please review it.